scripts/dorelease.py

```diff
@@ -105,7 +105,7 @@ def update_info(newver):
         infodict = json.load(infofile)
 
     verstring = infodict["VERSION"]
-    newverstring = re.sub("[0-9\.a-z]+", newver, verstring)
+    newverstring = re.sub(r"[0-9\.a-z]+", newver, verstring)
```

Comment:
Sligthly unrelated, but useful nonetheless, these r"" raw strings will fix a bunch of SyntaxWarnings which would stem to the failure to translate the escape code "\\." in this particular case.

Reply:
Yes, I always run `pyflakes` before submitting a PR; and the warnings were there, so ...

Reply:
Acknowledged, I guess that explains it and the programming languages selection in setup.py.  :)